### DISCUSSION TITLE
Collapsible objects/arrays

### DISCUSSION THREAD

**Discussion by ffxsam** [permafrost-dev/node-ray#154] [Ideas]
It would be great if, when we did something like this:

```js
ray({ someObj: { msg: 'Hello' } });
ray(superLongArrayOfComplexObjects);
```
that the object/array in the Ray UI would allow us to collapse/expand each object/array.

Thoughts on how this could be accomplished? (we're willing to contribute!)

CC @xndbogdan

**Comment by patinthehat** [comment]
@ffxsam @xndbogdan Thank you for offering to contribute! This is doable, but it'll take some work for sure.  I know how to do it, so I can walk you through the steps. DM or email me and we can have a longer discussion. :+1:

**Reply by xndbogdan** [reply]
I wish to contribute to the development efforts. I'll DM you, @patinthehat . Just let me know what needs to be done.
I really like what you did with this package.

**Comment by Sam-Apostel** [comment]
@ffxsam @xndbogdan `ray().toJson({ someObj: { msg: 'Hello' } });` should enable the collapsible ui in the Ray app.
